frontend/src/app/modules/auth/components/sign-up-form/sign-up-form.component.ts

```diff
@@ -33,18 +42,44 @@ export class SignUpFormComponent {
         confirmPassword: new FormControl('', { updateOn: 'blur' }),
     });
 
-    // eslint-disable-next-line no-empty-function
-    constructor(private authService: AuthService, private router: Router) {}
+    constructor(
+        private authService: AuthService,
+        private router: Router,
+        private userService: UserService,
+        private notifications: NotificationService,
+        // eslint-disable-next-line no-empty-function
```

Comment:
Please remove this

Reply:
But it's an empty constructor with injected services

Reply:
You need to extend BaseComponent and use super() here

Reply:
Fixed